.github/workflows/ios-unit-tests.yml

```diff
@@ -52,27 +56,25 @@ jobs:
         uses:                 actions-rs/install@v0.1
         with:
           crate:              uniffi_bindgen
-          version:            0.19.3 # Must be in sync with version in Cargo.toml
```

Comment:
The version and the comment were here for a reason.

Reply:
Fixed, sorry for that; removed trying to silence action warnings 🙏🏻